kafka-client/src/main/java/dev/responsive/kafka/internal/stores/PartitionedOperations.java

```diff
@@ -316,7 +317,7 @@ public byte[] get(final Bytes key) {
     return table.get(
         changelog.partition(),
         key,
-        minValidTimestamp()
+        currentRecordTimestamp()
```

Comment:
@agavra hey I was wondering if this should really be the current record timestamp or if streamTime is more accurate?

Or does it not really matter since we're approximating streamTime = currentTimestamp = wallclock time for the ttl semantics anyways